openid/pom.xml

```diff
@@ -125,19 +125,22 @@
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Bundle-Version>${project.version}</Bundle-Version>
                         <Export-Package>
+                            !org.eclipse.microprofile.*,
                             fish.payara.security.openid.*
                         </Export-Package>
                         <!-- Import API classes, other are NimbusDS optional dependencies -->
                         <Import-Package>
-                            javax.*, fish.payara.*, org.eclipse.*,
-                            *;resolution:=optional
+                            javax.*, fish.payara.*, !org.eclipse.microprofile.*,
+                            *;resolution:=optional, org.eclipse.microprofile.config;v
+ ersion="[1.0,3)"
```

Comment:
Oh, the end of the line was too heavy and it dropped ;)

Reply:
I missed that. I'll these on the same line.